jccli/jc_api_v1.py

```diff
@@ -37,7 +37,7 @@ def retrieve_users(self, user_ids=[]):
         Retrieve a list of users corresponding to ids
         """
         # FIXME: This is not an ideal way to do this, but search_systemusers_post doesn't seem to allow filtering on ID
-        all_users = self.get_users()
+        all_users = self.search_users()
```

Comment:
regarding the `FIXME` comment.  I am able to search for users with IDs like this..
```
curl -X POST https://console.jumpcloud.com/api/search/systemusers \
  -H 'Accept: application/json' \
  -H 'Content-Type: application/json' \
  -H 'x-api-key: {API_KEY} ' \
  -d '{
  "filter" : [{"_id" : "5d22bcd982d189675477b20c"}]
  }'
```

Reply:
Hmmm so I guess we do: `"filter" : [{"_username" : "some username here"}]` then? Should I do that in another PR, or open another issue?